ztp/gitops-subscriptions/argocd/example/policygentemplates/group-du-sno-ranGen.yaml

```diff
@@ -59,5 +59,48 @@ spec:
         disableDrain: true
     - fileName: StorageLV.yaml
       policyName: "local-disks-policy"
+      spec:
+        storageClassDevices:
+          - storageClassName: "example-storage-class-1"
+            volumeMode: Filesystem
+            fsType: xfs
+            devicePaths:
+              - /dev/sdb1
+          - storageClassName: "example-storage-class-2"
+            volumeMode: Filesystem
+            fsType: xfs
+            devicePaths:
+              - /dev/sdb2
     - fileName: DisableSnoNetworkDiag.yaml
       policyName: "config-policy"
+    - fileName: PerformanceProfile.yaml
```

Comment:
Moving this here will require the wave defined in PR 862 to be updated to match the other CRs in the group.

Reply:
Come to think of it, this MUST change because the multi-node group examples have the PerformanceProfile in their group PGTs.